code/mod_table/mod_table.cpp

```diff
@@ -162,16 +162,14 @@ static auto DiscordOption __UNUSED = options::OptionBuilder<bool>("Game.Discord"
                                if (!val) {
                                     Discord_presence = false;
                                     libs::discord::shutdown();
-                                    return true;
                                }
                           } else {
                                if (val) {
                                     Discord_presence = true;
                                     libs::discord::init();
-                                    return true;
                                }
                           }
-                          return false;
+                          return true;
                      })
```

Comment:
If it should always return true, should it return anything at all?

Reply:
Yes. .change_listener requires a book's return to know if an option successfully persisted or not. That tells it whether or not the option requires a restart.